packages/terra-date-picker/src/DatePicker.jsx

```diff
@@ -122,8 +122,8 @@ const defaultProps = {
   inputAttributes: undefined,
   isIncomplete: false,
   isInvalid: false,
-  maxDate: undefined,
-  minDate: undefined,
+  maxDate: DateUtil.MAX_DATE,
```

Comment:
Hmm, So is it acceptable to show default value as `DateUtil.MAX_DATE` instead of `2100-12-31` in props table.

<img width="1331" alt="Screen Shot 2020-06-01 at 5 00 16 PM" src="https://user-images.githubusercontent.com/12869809/83405577-677c4300-a42a-11ea-8a2a-c7354d4fc45c.png">

Reply:
I replaced with constants with the actual value. It looks like the actual value doesn't show in the prop table unless the constants are defined in the same file.
https://github.com/cerner/terra-framework/pull/1132/commits/50a1661d838b19bb8058fe44a7bfbc66212ec510